kinode/packages/terminal/terminal/src/lib.rs

```diff
@@ -396,3 +375,58 @@ fn handle_run(
 
     Ok(())
 }
+
+fn handle_alias_change(
+    state: &mut TerminalState,
+    alias: String,
+    process: Option<ProcessId>,
+) -> anyhow::Result<()> {
+    match process {
+        Some(process) => {
+            // first check to make sure the script is actually a script
+            let Ok(_) = get_entry(&process) else {
+                return Err(anyhow!("terminal: process {} not found", process));
```

Comment:
again, is this check necessary? it's not crazy to have aliases be settable for nonexistent scripts and show a descriptive error when the alias gets used

Reply:
I like having the extra check. It saves an extra step from `1. save alias, 2. try using alias and it fails, 3. reset alias` to just `1. get an error for failed alias 2. reset alias`

Plus this check adds basically no time